go/proto/billaged/v1/billing.proto

```
@@ -1,10 +1,10 @@
 syntax = "proto3";
 
-package deploy.billaged.v1;
+package billaged.v1;
```

Buf (1.55.1)

Files with package "billaged.v1" must be within a directory "billaged/v1" relative to root but were in directory "go/proto/billaged/v1".

(PACKAGE_DIRECTORY_MATCH)